features/dd-sdk-android-session-replay-compose/src/main/kotlin/com/datadog/android/sessionreplay/compose/internal/mappers/TabRowCompositionGroupMapper.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.compose.internal.mappers
+
+import com.datadog.android.sessionreplay.compose.internal.data.Box
+import com.datadog.android.sessionreplay.compose.internal.data.ComposableParameter
+import com.datadog.android.sessionreplay.compose.internal.data.ComposeWireframe
+import com.datadog.android.sessionreplay.compose.internal.data.UiContext
+import com.datadog.android.sessionreplay.model.MobileSegment
+import com.datadog.android.sessionreplay.utils.ColorStringFormatter
+
+internal class TabRowCompositionGroupMapper(
+    colorStringFormatter: ColorStringFormatter
+) : AbstractCompositionGroupMapper(colorStringFormatter) {
+    override fun map(
+        stableGroupId: Long,
+        parameters: Sequence<ComposableParameter>,
+        boxWithDensity: Box,
+        uiContext: UiContext
+    ): ComposeWireframe {
+        var backgroundColor: String? = null
+        var contentColor: String? = null
+        parameters.forEach { param ->
```

Comment:
isn't the case where you can have either the `backgroundColor` and `contentColor` available in the same time ?

Reply:
I am not sure what you mean here, do you have any suggestion?

Reply:
nevermind, i didn't see it well below :)